cardano-cli/src/Cardano/CLI/Shelley/Run/Node.hs

```diff
@@ -14,23 +15,69 @@ import           Cardano.Config.Shelley.OCert
 import           Cardano.Config.Shelley.VRF
 
 import           Cardano.Config.Shelley.ColdKeys (KeyError)
+import           Cardano.Config.TextView (textShow)
 import           Cardano.Config.Types (SigningKeyFile(..))
 
 import           Cardano.CLI.Shelley.Commands
 import           Cardano.CLI.Shelley.KeyGen
 
 
 data ShelleyNodeCmdError
-  = ShelleyNodeCmdKeyError !KeyError
-  | ShelleyNodeCmdOperationalCertError !OperationalCertError
-  | ShelleyNodeCmdReadKeyError !KeyError
-  | ShelleyNodeCmdKESError !KESError
-  | ShelleyNodeCmdVRFError !VRFError
-  | ShelleyNodeCmdKeyGenError !ShelleyKeyGenError
+  = ShelleyNodeReadStakePoolSignKeyError !FilePath !KeyError
+  | ShelleyNodeWriteOpCertIssueCounterError !FilePath !OperationalCertError
+  | ShelleyNodeWriteOperationalCertError !FilePath !OperationalCertError
+  | ShelleyNodeReadKESVerKeyError !FilePath !KESError
+  | ShelleyNodeWriteKESVerKeyError !FilePath !KESError
+  | ShelleyNodeWriteKESSignKeyError !FilePath !KESError
+  | ShelleyNodeWriteVRFSignKeyError !FilePath !VRFError
+  | ShelleyNodeWriteVRFVerKeyError !FilePath !VRFError
+  | ShelleyNodeReadOperationalCertCounterError !FilePath !OperationalCertError
+  | ShelleyNodeOperatorKeyGenError
+      !FilePath
+      -- ^ Verification key
+      !FilePath
+      -- ^ Signing key
+      !ShelleyKeyGenError
```

Comment:
Doesn't need to change, I just wonder whether this is more clear than using two `FilePath`s:

```suggestion
  | ShelleyNodeOperatorKeyGenError
      !VerificationKeyFile
      !SigningKeyFile
      !ShelleyKeyGenError
```

Reply:
:100: I think that is a change well worth making.